R/pkg/R/mllib.R

```diff
@@ -402,6 +406,8 @@ setMethod("spark.naiveBayes", signature(data = "SparkDataFrame", formula = "form
         return(new("NaiveBayesModel", jobj = jobj))
     })
 
+#' Save fitted MLlib model to the input path
```

Comment:
@jkbradley Likewise, I changed title `write.ml` to `Save fitted MLlib model to the input path` rather than `Save the Bernoulli naive Bayes model to the input path.` for all four different models.